cmd/server/assets/codestatus/show.html

```diff
@@ -18,41 +18,44 @@ <h1>Verification code status</h1>
       Status of a verification code previously shared with your patient.
     </p>
 
-    <div class="card mb-3">
+    <div class="card mb-3 shadow-sm">
       <div class="card-header">
         Code status
       </div>
-      <div class="list-group-item">
-        <h5 class="mb-1">UUID</h5>
-        <p class="mb-1 text-monospace">{{.code.UUID}}</p>
-      </div>
-      <div class="list-group-item">
-        <h5 class="mb-1">{{.code.IssuerType}}</h5>
-        <p class="mb-1">{{.code.Issuer}}</p>
-      </div>
-      <div class="list-group-item">
-        <h5 class="mb-1">Test type</h5>
-        <p class="mb-1">{{.code.TestType}}</p>
-      </div>
-      <div class="list-group-item">
-        <h5 class="mb-1">Status</h5>
-        <p class="mb-1">{{.code.Status}}</p>
-      </div>
-      <div class="list-group-item">
-        <h5 class="mb-1">Expiry</h5>
-        <span id="code-expires-at" class="sm text-danger"></span>
-      </div>
-      {{if .code.HasLongExpires}}
-      <div class="list-group-item">
-        <h5 class="mb-1">SMS link expiry</h5>
-        <span id="long-code-expires-at" class="sm text-danger"></span>
-      </div>
-      {{end}}
-      <div class="card-body">
-        <a href="/code/{{.code.UUID}}/expire" class="d-block text-danger" data-method="PATCH"
-          data-confirm="Are you sure you want to expire this code?" data-toggle="tooltip" title="Expire code">
-          <button type="submit" id="invalidate" class="btn btn-danger btn-sm">Invalidate code now</button>
-        </a>
+      <div class="list-group list-group-flush">
+        <div class="list-group-item">
+          <h5 class="mb-1">UUID</h5>
+          <p class="mb-1 text-monospace">{{.code.UUID}}</p>
+        </div>
+        <div class="list-group-item">
+          <h5 class="mb-1">{{.code.IssuerType}}</h5>
+          <p class="mb-1">{{.code.Issuer}}</p>
+        </div>
+        <div class="list-group-item">
+          <h5 class="mb-1">Test type</h5>
+          <p class="mb-1">{{.code.TestType}}</p>
+        </div>
+        <div class="list-group-item">
+          <h5 class="mb-1">Status</h5>
+          <p class="mb-1">{{.code.Status}}</p>
+        </div>
+        <div class="list-group-item">
+          <h5 class="mb-1">Expiry</h5>
+          <span id="code-expires-at" class="sm text-danger">&nbsp;</span>
```

Comment:
Empty elements have no height, so this created a weird FOUT on page load. Giving it a space solves that problem.